.github/CODEOWNERS

```diff
@@ -6,33 +6,44 @@
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-/**                      @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
+/**                      @rickwinter @LarryOsterman
+/sdk/                    @rickwinter @LarryOsterman
 
 # Samples
-/samples/                @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
+/samples/                @rickwinter @ahsonkhan @antkmsft @gearama @LarryOsterman
 
+# AzureSDKOwners: @gearama
+# ServiceLabel: %Azure.Core
 # PRLabel: %Azure.Core
-/sdk/core/               @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
+/sdk/core/               @rickwinter @ahsonkhan @antkmsft @gearama @LarryOsterman
 
+# AzureSDKOwners: @ahsonkhan
+# ServiceLabel: %Azure.Identity
 # PRLabel: %Azure.Identity
-/sdk/identity/           @antkmsft @schaabs @ahsonkhan @rickwinter @vhvb1989 @gearama @LarryOsterman @Azure/azure-sdk-write-identity
+/sdk/identity/           @antkmsft @schaabs @ahsonkhan @rickwinter @gearama @LarryOsterman @Azure/azure-sdk-write-identity
 
 ###########
 # Client SDKs
 ###########
 
+# AzureSDKOwners: @LarryOsterman
+# ServiceLabel: %Attestation
 # PRLabel: %Attestation
-/sdk/attestation/        @LarryOsterman @gkostal @anilba06 @ahmadmsft @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+/sdk/attestation/        @gkostal @anilba06 @ahmadmsft @LarryOsterman @rickwinter
 
+# AzureSDKOwners: @gearama
+# ServiceLabel:  %KeyVault
 # PRLabel: %KeyVault
 /sdk/keyvault/           @gearama @antkmsft @rickwinter @LarryOsterman
 
+# AzureSDKOwners: @gearama
```

Comment:
Do you have a doc or some pointer that explain what this comment now does (along with what `ServiceLabel')?

Does this mean an issue is auto-assigned to this person if is has that service label on it?

Reply:
The person is the first person to triage.  It doesn't mean they fix, but they are the first to look at it.